datastore/integration-test/response/items_get_single_platform.json

```diff
@@ -24,9 +24,9 @@
                 "institution": "Royal Netherlands Meteorological Institute (KNMI)",
                 "project": "",
```

Comment:
Why doesn't this test break? As the output is now without the empty strings, correct?

Reply:
I didn't touch the tests, only updated the test data. Seems like this is duplicate / older version of `items_within_area_single_platform.json` and is not used on any tests.